jwst/datamodels/schemas/slitdata.schema.yaml

```diff
@@ -0,0 +1,137 @@
+allOf:
+- type: object
+  properties:
+    name:
+      title: Name of the slit
+      type: string
+      fits_keyword: SLTNAME
+      fits_hdu: SCI
+    xstart:
+      title: Starting pixel in x direction
+      type: integer
+      default: 1
+      fits_keyword: SLTSTRT1
+      fits_hdu: SCI
+    xsize:
+      title: Number of pixels in x direction
+      type: integer
+      default: 0
+      fits_keyword: SLTSIZE1
+      fits_hdu: SCI
+    ystart:
+      title: Starting pixel in y direction
+      type: integer
+      default: 1
+      fits_keyword: SLTSTRT2
+      fits_hdu: SCI
+    ysize:
+      title: Number of pixels in y direction
+      type: integer
+      default: 0
+      fits_keyword: SLTSIZE2
+      fits_hdu: SCI
+    slitlet_id:
+      title: Slitlet ID
+      type: integer
+      default: 0
+      fits_keyword: SLITID
+      fits_hdu: SCI
+    source_id:
+      title: Source ID
+      type: integer
+      default: 0
+      fits_keyword: SOURCEID
+      fits_hdu: SCI
+    source_name:
+      title: Source name
+      type: string
+      fits_keyword: SRCNAME
+      fits_hdu: SCI
+    source_alias:
+      title: Source alias
+      type: string
+      fits_keyword: SRCALIAS
+      fits_hdu: SCI
+    stellarity:
+      title: Source stellarity
+      type: number
+      fits_keyword: STLARITY
+      fits_hdu: SCI
+    source_type:
+      title: Source type (point/extended)
+      type: string
+      fits_keyword: SRCTYPE
+      fits_hdu: SCI
+    source_xpos:
+      title: Source position in slit (x-axis)
+      type: number
+      default: 0.0
+      fits_keyword: SRCXPOS
+      fits_hdu: SCI
+    source_ypos:
+      title: Source position in slit (y-axis)
+      type: number
+      default: 0.0
+      fits_keyword: SRCYPOS
+      fits_hdu: SCI
+    shutter_state:
+      title: All (open and closed) shutters in a slit
+      type: string
+      default: ""
+      fits_keyword: SHUTSTA
+      fits_hdu: SCI
+    data:
+      title: The science data
+      fits_hdu: SCI
+      default: 0.0
+      ndim: 2
+      datatype: float32
+    dq:
+      title: Data quality array
+      fits_hdu: DQ
+      default: 0
+      datatype: uint32
+    err:
+      title: Error array
+      fits_hdu: ERR
+      default: 0.0
+      datatype: float32
+    wavelength:
+      title: Wavelength array, corrected for zero-point
+      fits_hdu: WAVELENGTH
+      ndim: 2
+      datatype: float32
+    barshadow:
+      title: Bar shadow correction
+      fits_hdu: BARSHADOW
+      ndim: 2
+      default: 1.0
+      datatype: float32
+    bunit_err:
```

Comment:
You forgot to remove bunit_err, which is now already included in bunit.schema.yaml

Reply:
I didn't realize it was included in `bunit`. Nice.